test.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+    <TargetPlatform>x86</TargetPlatform>
+  </RunConfiguration>
+</RunSettings>
```

Comment:
This should probably be gitignored and not checked in.

Reply:
I'll add *.runsettings to .gitignore